wvd-templates/DSC/Script-CreateHostPool.ps1

```diff
@@ -92,7 +92,7 @@ if ($HostPool) {
 Write-Log -Message "$HostpoolName Hostpool does not exist in $TenantName Tenant"
 
 $HostPool = New-RdsHostPool -TenantName "$TenantName" -Name "$HostPoolName" -Description $Description -FriendlyName $FriendlyName -Persistent:($EnablePersistentDesktop -ieq "true") -ValidationEnv $false
-$HName = $HostPool.name | Out-String -Stream
+$HName = $HostPool.HostPoolName | Out-String -Stream

```

Comment:
Addressing: Script-FirstRdshServer.ps1 - Hostpool name is not present in ScriptLog file #322